.gitignore

```diff
@@ -1,2 +1,4 @@
 source1/.vscode/settings.json
 source1/.vscode/tasks.json
+.idea
```

Comment:
Apparently you did add `.idea` and `.vscode` to gitignore, but didn't remove them. Use `git rm --cached [file name]` and `git rm -r --cached [directory name]` to remove them